### PR TITLE
Code Cleanup

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [ ] New feature
- [X] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

Remove unused parts and remove empty new lines from the rendered templates.

### CODE REVIEW

**Review by jonandernovella — DISMISSED** [review]
LGTM

File: ega-charts/localega/values.yaml [hunk 1]
```diff
@@ -156,7 +156,7 @@ dataedge:
   servicePort: 9059
   debug: 5058
   logpath: "/tmp/logs"
-  keystorePass: "dataedge"
+  keystorePass: "changeit"
```
Comment (blankdots) [inline ega-charts/localega/values.yaml:159]: if we are going to use one, why not set it once in the `secrets` ?

**Review by blankdots — COMMENTED** [review]
Should we make a release with this one?

File: ega-charts/localega/values.yaml (hunk 1 above)
Comment (jbygdell) [reply ega-charts/localega/values.yaml:159]: That password is set when you run `deploy-init` the default value is `changeit`.
It is needed in order to open and use the keystore.

**Comment by jbygdell** [comment]
> Should we make a release with this one?

After this is merged we can tag a new release.

File: ega-charts/localega/values.yaml (hunk 1 above)
Comment (blankdots) [reply ega-charts/localega/values.yaml:159]: let us leave for each service to use different keystore passwords

**Review by blankdots — DISMISSED** [review]

**Review by viklund — APPROVED** [review]

**Review by blankdots — APPROVED** [review]

**Pull request merged**